mods/ra/weapons/smallcaliber.yaml

```diff
@@ -217,11 +217,15 @@ ChainGun:
 		Versus:
 			None: 144
 
-ChainGun.Yak:
+ChainGun.Yak.Left:
 	Inherits: ^HeavyMG
-	ReloadDelay: 3
+	Burst: 5
+	FirstBurstTargetOffset: -1068,-213,0
+	FollowingBurstTargetOffset: 534,0,0
+	TargetActorCenter: true
+	BurstDelay: 3
```

Comment:
BurstDelay*s*.

Reply:
I don't think this impacts the behaviour of the testcase, so i'll leave this as an exercise to the reader before we drop the commit :smile: